x/auth/client/cli/tip.go

```diff
@@ -0,0 +1,84 @@
+package cli
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	clienttx "github.com/cosmos/cosmos-sdk/client/tx"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/cosmos/cosmos-sdk/types/tx"
+	authclient "github.com/cosmos/cosmos-sdk/x/auth/client"
+	authsigning "github.com/cosmos/cosmos-sdk/x/auth/signing"
+)
+
+func GetTipsToFeeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "tips-to-fee <aux_signed_tx.json>",
```

Comment:
Since we're calling AddAuxSignerData, maybe we can rename this to `add-aux`. Then, if there are multiple AddAuxSignerDatas, the fee payer can call this command multiple times